### DISCUSSION TITLE
Is STOP event supposed to trigger twice?

### DISCUSSION THREAD

**Discussion by mjau-mjau** [simonwep/viselect#225] [Q&A] [answered]
Since version 3.2.0, it seems the stop event triggers twice on single click select. In version 3.1.0, it triggered only once.

See Codepen:
https://codepen.io/mjau-mjau/pen/gOydNBX

Is this expected behavior, or am I missing something?

**Comment by simonwep** [comment, marked as answer]
No, that looks like a bug to me - could you create an issue for that? I'll soon work on open issues again :)
